apps/design/backend/src/worker/generate_election_package_and_ballots.ts

```diff
@@ -191,11 +188,7 @@ export async function generateElectionPackageAndBallots(
     JSON.stringify(systemSettings, null, 2)
   );
 
-  const isCloudTranslationAndSpeechSynthesisEnabled =
-    isVxOrSliOrg(orgId) &&
-    isFeatureFlagEnabled(
-      BooleanEnvironmentVariableName.ENABLE_CLOUD_TRANSLATION_AND_SPEECH_SYNTHESIS
-    );
+  const isCloudTranslationAndSpeechSynthesisEnabled = shouldExportAudio;
   const { uiStringAudioIds, uiStringAudioClips } = generateAudioIdsAndClips({
     isCloudTranslationAndSpeechSynthesisEnabled,
```

Comment:
Should we just omit the audio files completely if `shouldExportAudio` is false rather than returning an empty file?

Reply:
Caller logic simplified [here](https://github.com/votingworks/vxsuite/pull/6088/commits/5687aeede41513924cc158d22047fa2e6e4007ac#diff-c268bf2996342639e8bfd885b518b620095551a7b47c8517f93bab768c57bfd0R191) and param removed [here](https://github.com/votingworks/vxsuite/pull/6088/commits/5687aeede41513924cc158d22047fa2e6e4007ac#diff-1022e77fe3d00d85799195dd201761ed822cfc114b1548a88587b1549dbf5ec1L39).